### DISCUSSION TITLE
How build this project using musl compiler

### DISCUSSION THREAD

**Discussion by loss-and-quick** [Floorp-Projects/Floorp#289] [Q&A] [answered]
I want to install this browser on void linux

**Comment by Ryosuke-Asano** [comment]
Did you try to use Firefox? I want to know about Firefox on void Linux

**Comment by legobeat** [comment, marked as answer]
Firefox does compile using musl and runs on Void Linux.

Sources for the Firefox package in the Void repos are [here](https://github.com/void-linux/void-packages/blob/master/srcpkgs/firefox/template). @loss-and-quick Maybe you could try starting with Firefox as a base for your build. [Pointers here](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md)

Similar story for [Alpine](https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/firefox/APKBUILD).

**Comment by Ryosuke-Asano** [comment]
Floorp can build like as Firefox. try @legobeat's comment.
